File: api-stockhub/stockhub/src/main/java/com/api/stockhub/models/ColaboradorModel.java
package com.api.stockhub.models;

import java.io.Serializable;

import jakarta.persistence.Entity;
import jakarta.persistence.Id;
import jakarta.persistence.Table;

@Entity
@Table(name = "TB_COLABORADOR")
public class ColaboradorModel implements Serializable {
	private static final long serialVersionUID = 1L;

	@Id
	private Long matricula;

	private String nome;

	private String empresa;

	private String licenca;

	private String funcao;

	private String setor;
	
	

	public Long getMatricula() {
		return matricula;
	}

	public void setMatricula(Long matricula) {
		this.matricula = matricula;
	}

	public String getNome() {
		return nome;
	}

	public void setNome(String nome) {
		this.nome = nome;
	}

	public String getEmpresa() {
		return empresa;
	}

	public void setEmpresa(String empresa) {
		this.empresa = empresa;
	}

	public String getLicenca() {
		return licenca;
	}

	public void setLicenca(String licenca) {
		this.licenca = licenca;
	}

	public String getFuncao() {
		return funcao;
	}

	public void setFuncao(String funcao) {
		this.funcao = funcao;
	}

	public String getSetor() {
		return setor;
	}

	public void setSetor(String setor) {
		this.setor = setor;
	}

	public static long getSerialversionuid() {
		return serialVersionUID;
	}

}

File: Api/Api/Domain/Models/ColaboradorModel.cs
namespace Api.Domain.Models
{
    public class ColaboradorModel : Base
    {
        public long Matricula { get; set; }
        public string Nome { get; set; }
        public string Empresa { get; set; }
        public string Licenca { get; set; }
        public string Funcao { get; set; }
        public string Setor { get; set; }
        public string Usuario { get; set; }
        public string Senha { get; set; }

        public List<MovimentacaoModel> Movimentacao { get; set; }
    }
}
